README.md

```diff
@@ -9,6 +9,12 @@ Pigment CSS is a zero-runtime CSS-in-JS library that extracts the colocated sty
 
 <div align="center">
 
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=0ckOUBiuxVY&t=21677s" target="_blank"><img src="https://img.youtube.com/vi/0ckOUBiuxVY/0.jpg" alt="Introduction of Pigment CSS during React Conf 2024" style="width: 50%; height: 50%; aspect-ratio: 16/9;" /></a>
```

Comment:
```suggestion
Check out Olivier Tassinari introducing Pigment CSS at React Conf 2024:

<a href="http://www.youtube.com/watch?feature=player_embedded&v=0ckOUBiuxVY&t=21677s" target="_blank"><img src="https://img.youtube.com/vi/0ckOUBiuxVY/0.jpg" alt="Introduction of Pigment CSS during React Conf 2024" style="width: 50%; height: 50%; aspect-ratio: 16/9;" /></a>
```

Same reasoning from here → https://github.com/mui/material-ui/pull/42500#discussion_r1624595201 — but just omitting "MUI's CEO" because it can be a little weird for a README file.